net.hachune.mikutter.appdata.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Yuto Tokunaga <yuntan.sub1@gmail.com> -->
+<component type="desktop">
+  <id>net.hachune.mikutter</id>
+  <metadata_license>MIT</metadata_license>
+  <name>mikutter</name>
+  <name xml:lang="ja">mikutter</name>
+  <summary>Simple, powerful and moeful twitter client</summary>
+  <summary xml:lang="ja">至高のTwitterクライアント</summary>
+  <icon type="remote" height="256" width="256">https://github.com/mikutter/mikutter/raw/master/core/skin/data/icon.png</icon>
+  <description>
+    <p>mikutter is a twitter client.</p>
+    <p xml:lang="ja">全てのミク廃、そしてTwitter中毒者へ贈る、至高のTwitter Clientを目指すTwitter Client。略して至高のTwitter Client。</p>
+  </description>
+  <categories>
+    <category>Network</category>
+    <category>Feed</category>
+    <category>GTK</category>
+  </categories>
+  <url type="homepage">https://mikutter.hachune.net</url>
+  <url type="bugtracker">https://dev.mikutter.hachune.net/projects/mikutter/</url>
+  <url type="faq">https://mikutter.hachune.net/faq</url>
+  <url type="help">https://yuzuki.hachune.net/wiki/FrontPage</url>
+  <launchable type="desktop-id">net.hachune.mikutter.desktop</launchable>
+  <releases>
+    <release version="3.7.1" date="2018-05-13">
+      <description>
+        <p>Release note: http://mikutter.hatenablog.com/entry/2018/05/13/022743</p>
+      </description>
+    </release>
+    <release version="3.7.0" date="2018-05-07">
+      <description>
+        <p>Release note: http://mikutter.hatenablog.com/entry/2018/05/07/010457</p>
+      </description>
+    </release>
+  </releases>
+  <provides>
+    <binary>mikutter</binary>
+  </provides>
+  <project_license>MIT</project_license>
+  <developer_name>Toshiaki Asai</developer_name>
+  <screenshots>
+    <screenshot type="default">
+      <image width="500" height="299" xml:lang="ja">https://mikutter.hachune.net/img/mikutter-normal.png</image>
+      <caption>Multipane</caption>
+      <caption xml:lang="ja">マルチペイン</caption>
+    </screenshot>
+    <screenshot>
+      <image width="427" height="284" xml:lang="ja">https://mikutter.hachune.net/img/mikutter-multi-account.png</image>
+      <caption>Multi account</caption>
+      <caption xml:lang="ja">マルチアカウント</caption>
+    </screenshot>
+  </screenshots>
+  ⁠<update_contact>yuntan.sub1_AT_gmail.com</update_contact>
+  <translation type="gettext">mikutter</translation>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">none</content_attribute>
```

Comment:
If you use `oars-1.1` you can remove all the `content_attribute` tags with `none` value :sunglasses:

Reply:
Thanks 🙂